src/main/ml-modules/services/geoQueryService.sjs

```diff
@@ -11,6 +11,8 @@ const sql2optic = require('/ext/sql/sql2optic.sjs');
 const geostats = require('/ext/geo/geostats.js');
 const geoextractor = require('/ext/geo/extractor.sjs');
 const qd = require('/ext/query/ctsQueryDeserialize.sjs').qd;
+const sq = require("/ext/query/structured-query-utils.xqy");
+const search = require("/MarkLogic/appservices/search/search.xqy");
```

Comment:
Since this is an internal MarkLogic library, does this introduce any dependency on a specific version of MarkLogic?

Reply:
Maybe we can move this out of this module and keep all the internal dependencies in the utils lib?

Reply:
Part of the UI is leaning on /v1/search. How can we ensure they behave the same if we are not using the same code underneath?

Reply:
search is a public interface by the way.. https://docs.marklogic.com/search

Reply:
Oh yeah, sorry, I read that too fast. The search:search API is fine.